utils/sane-backends/Makefile

```diff
@@ -130,6 +130,20 @@ $(call Package/sane-backends/Default)
   USERID=saned:saned
 endef
 
+define Package/sane-daemon-avahi-dbus-service
+$(call Package/sane-backends/Default)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS+=sane-daemon +avahi-dbus-daemon
+  TITLE+= (network daemon avahi service files)
+endef
+
+define Package/sane-daemon-avahi-nodbus-service
```

Comment:
As far as I can see, there is no difference between dbus and nodbus.
Normally you have different packages when the binary is linked to different equivalent libraries (i.e. *ssl).
If it is just a way to select different dependencies, it would be interesting to have this solved in DEPENDS.
I guess both provides avahi-daemon.